api-gateway/src/main/resources/application-internal.yml

```diff
@@ -0,0 +1,50 @@
+gateway:
+  backend:
+    url: http://localhost:5800
```

Comment:
no need to duplicate these properties from `dev`. we can enable two profiles at the same time: `dev,internal`.
I think need to leave only: `spring.security.oauth2.client.provider.codehub.*` and `spring.security.oauth2.client.registration.codehub.*`

Reply:
deleted all unnecessary properties so that we can use only `internal` profile in HIS cloud